dproc/dproc.go

```diff
@@ -71,6 +71,16 @@ func GetDataprocRequestParams(sourceProfile profiles.SourceProfile, targetProfil
 			jdbcParams["sql"] = fmt.Sprintf("select * from %s.%s", srcSchema, srcTable)
 		}
 		jdbcParams["jar"] = "gs://dataproc-templates/jars/mysql-connector-java.jar"
+	} else if sourceProfile.Driver == constants.POSTGRES {
+		jdbcParams["url"] = fmt.Sprintf("jdbc:postgresql://%s:%s/%s?user=%s&password=%s", host, port, srcDb, user, pwd)
+		jdbcParams["driver"] = "org.postgresql.Driver"
+		if primaryKeys == "" {
+			jdbcParams["sql"] = fmt.Sprintf("select gen_random_uuid() as synth_id, t.* from %s.%s as t", srcSchema, srcTable)
+			primaryKeys = "synth_id"
+		} else {
+			jdbcParams["sql"] = fmt.Sprintf("select * from %s.%s", srcSchema, srcTable)
+		}
+		jdbcParams["jar"] = "gs://dataproc-templates/jars/postgresql-42.2.6.jar"
```

Comment:
it seems like every time there's a release this will need to be changed. Can we have a common location where the release version will be updated so that it gets picked up automatically without code change in HB ?
It seems like the mysql-connector is doing that.

Reply:
Valid Point!
Do we have any public bucket for harbourbridge or spanner, which we can reuse.
I'll also check with dataproc team, if they have any alternatives.

Reply:
Spoke with Surjit about this; so the jar file for the database connector should actually be supplied by the user. The expectation is they download whatever connector jar file they need and reference that as they run dataproc template. This also applies to mysql-connector jar file. The one we were referencing earlier is not guaranteed to be the most up to date.

Reply:
We should probably add another input box in the Dataproc config to allow user to pass in this parameter.